analysers/analyser_osmosis_indoor.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Noémie Lehuby 2022                                         ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from modules.OsmoseTranslation import T_
+from .Analyser_Osmosis import Analyser_Osmosis
+
+sql10 = """
+SELECT
+  id,
+  ST_AsText(way_locate(linestring))
+FROM
+  ways
+WHERE
+  NOT is_polygon
+  AND tags?'indoor'
+  AND tags->'indoor' in ('room', 'corridor', 'area', 'level')
+"""
+
+sql20 = """
+SELECT
+  id,
+  ST_AsText(geom)
+FROM
+  nodes
+WHERE
+  tags?'indoor'
+  AND tags->'indoor' in ('room', 'corridor', 'area', 'level')
+"""
```

Comment:
Can be done using sax plugin (faster), from python or MapCSS code.

Reply:
I know, but I thought it wouldn't make sense to create a plugin only for 1 or 2 checks if we already have an osmosis analysis.
Do you want me to make it mapcss instead ?